packages/devextreme/js/__internal/ui/html_editor/modules/m_resizing.ts

```diff
@@ -207,7 +207,14 @@ export default class ResizingModule extends BaseModule {
 
     if (option === 'enabled') {
       this.enabled = value;
-      value ? this._attachEvents() : this._detachEvents();
+      if (value) {
+        this._attachEvents();
+        if (!this.resizable) {
+          this._createResizeFrame();
+        }
+      } else {
+        this._detachEvents();
```

Comment:
Now resizeFrame is remaining in DOM after we disable resizing, but it's supposed to be removed.
```suggestion
        this.clean();
```

Reply:
[602a382](https://github.com/DevExpress/DevExtreme/pull/27707/commits/602a382a5a6857d3aff06cf12b52c3b8a00cc04b)